### ISSUE TITLE
Make sure to create your project repo

### ISSUE THREAD

**Issue by bensoltoff** [UC-MACSS/persp-research_Spr18#4]
If you don't follow the instructions in the README.md exactly as we specify, we will not be able to find your presentations and you will not present your research topic tomorrow.

**Comment by ChenAnhua** [comment]
Thanks for the clarification! Do we also need to send the link of our proposal as an issue on this class repo?

**Comment by bensoltoff** [comment]
No. Just make sure you create it using the same GitHub account you used in past terms and give it exactly the name we said

**Comment by ChenAnhua** [comment]
Thanks!

**Comment by siyiii** [comment]
Hi Benjamin,


I didn’t register for this course. Thanks!


Best,
Siyi

2018年4月4日星期三，Benjamin Soltoff <notifications@github.com> 写道：

**Comment by Shuting05** [comment]
Hi,Benjamin,
This is my project repo,
https://github.com/Shuting05/MACS30200proj
I created it two days ago but can you find it now? Thanks!

**Comment by bensoltoff** [comment]
Yes, I had the wrong GitHub username.


~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Dr. Benjamin Soltoff
209 McGiffert House
Lecturer in Computational Social Science
University of Chicago
~~~~~~~~~~~~~~~~~~~~~~~~~~~~

**Comment by RuxinChen** [comment]
I created a new repo: 
https://github.com/RuxinChen/MACS30200proj
